app/realtime.php

```diff
@@ -32,41 +41,81 @@
 ini_set('default_socket_timeout', -1);
 Swoole\Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
 
-$server = new Server("0.0.0.0", 80);
+$server = new Server('0.0.0.0', 80);
+
+$server->set([
+    'websocket_compression' => true,
+    'package_max_length' => 81920
```

Comment:
Is this bytes? please add a comment for easy reference

Reply:
Yes, will do 👍🏻 

Just added this as some random value to have payload limitation for now.